documentation20/cn/02.getting-started/02.taosdemo/docs.md

```diff
@@ -145,7 +145,7 @@ insert delay, avg:       8.31ms, max:     860.12ms, min:       2.00ms
 ```
 $ taosdemo --help
 
--f, --file=FILE The meta file to the execution procedure.
+-f, --file=FILE The meta file to the execution procedure. Recommend UTF-8 instead of UTF-8 with BOM.
```

Comment:
Currently, we support standard UTF-8 (without BOM) encoded files only.

Reply:
ok